frontend/public/index.html

```diff
@@ -14,7 +14,7 @@
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <!-- <link rel="manifest" href="%PUBLIC_URL%/manifest.json" /> -->
```

Comment:
Removed as we don't have a `manifest.json`

Reply:
Good catch!

As a follow-up, I see a few things related to this:
- Update the "description" meta tag
- Maybe, we should remove the `apple-touch-icon` and the related image file
- We should update the Favicon

Reply:
I messed sth up here by writing an incorrect deployment script. Undo upcoming.